src/transformers/models/data2vec/configuration_data2vec.py

```diff
@@ -115,6 +115,22 @@ def __init__(
         position_embedding_type="absolute",
         use_cache=True,
         classifier_dropout=None,
+        # Here starts audio-only parameters
```

Comment:
We can refactor this to make a clear distinction between common parameters and modality-specific parameters

Reply:
Good for now!